src/perspective_view/panel.ts

```diff
@@ -832,17 +944,40 @@ export class PerspectivePanel extends RenderedDataPanel {
     );
 
     if (hasTransparent) {
-      // Draw transparent objects.
-      gl.depthMask(false);
-      gl.enable(WebGL2RenderingContext.BLEND);
+      //Draw transparent objects.
+      const { maxProjectionConfiguration } = this;
+      const bindMaxProjectionBuffer = () => {
```

Comment:
We could add a `hasMaxProjection` variable so that we don't create/clear the max projection framebuffers if not needed.